src/AbsoluteHack/templates/AbsoluteHackInit/index.html

```diff
@@ -5,18 +5,18 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no" />
 	<title><%= htmlWebpackPlugin.options.title %></title>
-
-	<%= require('./_meta.ejs')({ data: htmlWebpackPlugin.options }) %>
-	<%= require('./_favicons.ejs')() %>
+	{% load static %}
```

Comment:
Revert all changes in the code of this file. Just modify the static files' paths only, as per your project structure.

Reply:
thats exactly what I did

Reply:
This would get rendered to (for eg.) require('/static/ejs/_meta.ejs') where meta.ejs is stored

Reply:
> This would get rendered to (for eg.) require('/static/ejs/_meta.ejs') where meta.ejs is stored

Did u test the npm build with these changes?

Reply:
configured!
works for now.